src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/customer_group/create.yaml

```diff
@@ -0,0 +1,24 @@
+twig_hooks:
+    hooks:
+        'sylius_admin.customer_group.create.content':
+            form:
+                component: 'sylius_admin:customer_group:form'
```

Comment:
I wonder if, for such a simple resource, we need live components? 🤔

Reply:
I believe it's necessary if we want to have dynamic validation on the form.

Reply:
I understand your concerns. Perhaps we should consider adding a generic LiveComponent to cover these basic resources, wdyt?

Reply:
It sounds interesting 🤔 But it could be done in a separate PR, especially that there is some other resources to cover that way